### ISSUE TITLE
Using KML Regions to Display Large  Datasets 

### ISSUE THREAD

**Issue by QingMings** [NASAWorldWind/WebWorldWind#84]
The current implementation is to load all images at once, not  processing LOD .it should level by level  load。

**Comment by balhar-jakub** [comment]
Yes It currently doesn't support LOD though it does support Region with min altitude and max altitude. 
Do you have any example KML, you would like to display? I would take a look at what is possible to do. Add support for this shouldn't be very difficult.

**Comment by QingMings** [comment]
Tomorrow, I'm looking for a KML file  ：）,(It's on the company computer，  Time is already  2：08 AM)。

I hope you can help me ,support Lod .:)

**Comment by QingMings** [comment]
hi! @balhar-jakub  , I uploaded it.
[cs1.kmz](https://github.com/QingMings/kmlupload/blob/master/cs1.kmz)

It's a little big. If you have any questions, please contact me

**Comment by AkeluX** [comment]
@balhar-jakub any feedback on this?

**Comment by balhar-jakub** [comment]
It seems that we aren't going to support Lod in the near future. For now altitude is supported for the use of loading via Regions. 
I will try to create separately some workaround, which will work at least for the most common screens and provide it outside of the core of the framework. 
Just for your information: If you wanted to update how any of the KML classes are handled, simply subclass them and via KmlElements.addKey(KmlRegion.prototype.getTagNames()[0], KmlRegion); replace the used object with a new one. In this case it would be a subclass of KmlRegion and the updated function KmlRegion.prototype.intersectsVisible which will take Lod into account on top od the altitude and area.
